src/OrbitGl/FormatCallstackForTooltipTest.cpp

```diff
@@ -120,33 +120,39 @@ TEST(CallstackTooltipUtils, FormatCallstackForTooltipShortensLongCallstacks) {
 
   constexpr const char* kFunction12 = "void bazbaz(int,int)";
   constexpr const char* kModulePath2 = "/path/to/module2";
-  constexpr const char* kModuleName2 = "module2";
   orbit_client_data::LinuxAddressInfo address_info12{kFrame12, kOffsetInFunction, kModulePath2,
                                                      kFunction12};
   capture_data.InsertAddressInfo(address_info12);
 
   ModuleManager module_manager{};
   std::string formatted_callstack = FormatCallstackForTooltip(
-      empty_callstack, capture_data, module_manager, std::numeric_limits<size_t>::max(), 6, 2);
-
-  std::vector<std::string> expected_formatted_callstack{};
-  expected_formatted_callstack.push_back(absl::StrCat(kModuleName, " | ", kFunction1, "<br/>"));
-  expected_formatted_callstack.push_back(absl::StrCat(kModuleName, " | ", kFunction2to10, "<br/>"));
-  expected_formatted_callstack.push_back(absl::StrCat(kModuleName, " | ", kFunction2to10, "<br/>"));
-  expected_formatted_callstack.push_back(absl::StrCat(kModuleName, " | ", kFunction2to10, "<br/>"));
-  expected_formatted_callstack.push_back(
-      absl::StrCat("<i>... shortened for readability ...</i><br/>"));
-  expected_formatted_callstack.push_back(absl::StrCat(kModuleName, " | ", kFunction11, "<br/>"));
-  expected_formatted_callstack.push_back(absl::StrCat(kModuleName2, " | ", kFunction12, "<br/>"));
-  EXPECT_EQ(formatted_callstack, absl::StrJoin(expected_formatted_callstack, ""));
+      callstack, capture_data, module_manager, std::numeric_limits<size_t>::max(), 6, 2);
+
+  constexpr const char* kExpectedModuleName2 = "module2";
+
+  std::string expected_formatted_callstack{};
+  {
+    std::vector<std::string> expected_formatted_callstack_frames{
+        absl::StrCat(kModuleName, " | ", kFunction1, "<br/>"),
+        absl::StrCat(kModuleName, " | ", kFunction2to10, "<br/>"),
+        absl::StrCat(kModuleName, " | ", kFunction2to10, "<br/>"),
+        absl::StrCat(kModuleName, " | ", kFunction2to10, "<br/>"),
+        absl::StrCat("<i>... shortened for readability ...</i><br/>"),
+        absl::StrCat(kModuleName, " | ", kFunction11, "<br/>"),
+        absl::StrCat(kExpectedModuleName2, " | ", kFunction12, "<br/>"),
```

Comment:
After reading that we have `kModuleName`, I regret about my suggestion. Maybe come back to `kModuleName2`

Reply:
Done